.changeset/green-penguins-press.md

```diff
@@ -0,0 +1,36 @@
+---
+'@contentful/f36-accordion': patch
+'@contentful/f36-asset': patch
+'@contentful/f36-autocomplete': patch
+'@contentful/f36-badge': patch
+'@contentful/f36-button': patch
+'@contentful/f36-card': patch
+'@contentful/f36-collapse': patch
+'@contentful/f36-copybutton': patch
+'@contentful/f36-datetime': patch
+'@contentful/f36-drag-handle': patch
+'@contentful/f36-entity-list': patch
+'@contentful/f36-forms': patch
+'@contentful/f36-icon': patch
+'@contentful/f36-icons': patch
+'@contentful/f36-list': patch
+'@contentful/f36-menu': patch
+'@contentful/f36-modal': patch
+'@contentful/f36-note': patch
+'@contentful/f36-notification': patch
+'@contentful/f36-pagination': patch
+'@contentful/f36-pill': patch
+'@contentful/f36-popover': patch
+'@contentful/f36-skeleton': patch
+'@contentful/f36-spinner': patch
+'@contentful/f36-table': patch
+'@contentful/f36-tabs': patch
+'@contentful/f36-text-link': patch
+'@contentful/f36-tooltip': patch
+'@contentful/f36-typography': patch
+'@contentful/f36-workbench': patch
+'@contentful/f36-core': patch
+'@contentful/f36-components': patch
+---
+
+bump pacakges versions to have consisntent versioning
```

Comment:
This text is generated, right?

but it has some typos 👀 

```suggestion
bump packages versions to have consistent versioning
```

Reply:
actually no, I made the typo hahah
When you run `yarn changeset` you need to put a message about what is the version bump about